manifests/user.pp

```diff
@@ -22,6 +22,7 @@
                       },
   $ensure = present,
   $recurse_permissions = false,
+  $authorized_keys_file = "${home_dir}/.ssh/authorized_keys",
```

Comment:
the home_dir var is not able to used in the parameters because its not be declared yet.

This logic will need to be done inside the manifest.